### PR TITLE
Add initial prompt and language parameters to transcript function

### PR DESCRIPTION
**Pull request by AllenDang**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -22,7 +22,7 @@ serde = { version = "1.0.203", features = ["derive"] }
 serde_json = "1.0.117"
 strum = { version = "0.26.2", features = ["derive"] }
 ureq = "2.9.7"
-whisper-rs = { version = "0.11.1", features = ["whisper-cpp-log"] }
+whisper-rs = { git = "https://github.com/tazz4843/whisper-rs", branch = "master" }
```
Comment (sigaloid) [inline Cargo.toml]: Can't use the git repo since I publish to crates.io, this should be replaced with just the newest version of whisper-rs

**Review by sigaloid — CHANGES_REQUESTED** [review]
Looks good to me otherwise!

**Comment by sigaloid** [comment]
Side note: Is the whisper-cpp-log feature no longer used in 0.11? I noticed you took that feature out

**Comment by AllenDang** [comment]
> Side note: Is the whisper-cpp-log feature no longer used in 0.11? I noticed you took that feature out

Oh, I must missed that, will add it back.

**Review by sigaloid — APPROVED** [review]

**Pull request merged**